HoloPipelines/pipelines/components/compSeparateNumpy.py

```diff
@@ -0,0 +1,54 @@
+# this comp generate numpy lists list from unique value from the numpy list input (comp exclusive for abdomen model at the moment)
```

Comment:
Yes, you're right, this component is not involved in the sitk replacement. I'll remove it.

Reply:
If the undesired changes were introduced in a specific commit, you can do a [git revert](https://www.atlassian.com/git/tutorials/undoing-changes/git-revert).

(PS: Without knowing the details, I think I prefer components with the same I/O for all, and making strict constraints rather than accepting a broad range of possible I/O)